src/Data/Vec/Bounded.agda

```diff
@@ -10,9 +10,8 @@ module Data.Vec.Bounded where
 
 open import Level using (Level)
 open import Data.Nat.Base
-open import Data.Vec.Base using (Vec)
-import Data.Vec as Vec using (filter; takeWhile; dropWhile)
-open import Function
+open import Data.Vec as Vec using (Vec)
```

Comment:
This looks like a regression, i.e. an import of a super-module where a sub-module used to be used?